config/vufind/CookieConsent.yaml

```diff
@@ -0,0 +1,132 @@
+          Domain: '{{vufind_cookie_domain}}'
+          Description: CookieConsent::cookie_description_search_open_html
+          Expiration: session
+    matomo:
+      Title: CookieConsent::analytics_cookies_title_html
+      Description: CookieConsent::analytics_cookies_description_html
+      DefaultEnabled: false
+      Essential: false
+      ControlVuFindServices:
+        - matomo
+      Cookies:
+        - Name: _pk_id.*
+          Domain: '{{current_host_name}}'
+          Description: CookieConsent::cookie_description_matomo_id_html
+          Expiration: 13
+          ExpirationUnit: months
+        - Name: _pk_ref.*
+          Domain: '{{current_host_name}}'
+          Description: CookieConsent::cookie_description_matomo_ref_html
+          Expiration: 6
+          ExpirationUnit: months
+        - Name: _pk_ses.*, _pk_cvar.*, _pk_hsr.*
+          Domain: '{{current_host_name}}'
+          Description: CookieConsent::cookie_description_matomo_ses_html
+          Expiration: session
+      AutoClearCookies:
+        - Name: '/^_pk_/'
+    video_social_media:
+      Title: CookieConsent::video_social_media_cookies_title_html
+      Description: CookieConsent::video_social_media_cookies_description_html
+      DefaultEnabled: false
+      Essential: false
+      ControlIframeServices:
+        - youtube
+      Cookies:
+        - Name: yt-player-bandwidth
+          Domain: youtube.com
+          ThirdParty: true
+          Description: CookieConsent::cookie_description_third_party_bandwidth_html
+          Expiration: session
+        - Name: yt-player-headers-readable
+          Domain: youtube.com
+          ThirdParty: true
+          Description: CookieConsent::cookie_description_third_party_preferences_html
+          Expiration: session
+        - Name: GPS
+          Domain: youtube.com
+          ThirdParty: true
+          Description: CookieConsent::cookie_description_third_party_gps_html
+          Expiration: session
+        - Name: VISITOR_INFO1_LIVE
+          Domain: youtube.com
+          ThirdParty: true
+          Description: CookieConsent::cookie_description_third_party_bandwidth_html
+          Expiration: 6
+          ExpirationUnit: months
+        - Name: PREF, __Secure-YEC
+          Domain: youtube.com
+          ThirdParty: true
+          Description: CookieConsent::cookie_description_third_party_preferences_html
+          Expiration: 8
+          ExpirationUnit: months
+        - Name: YSC, CONSENT, SOCS
+          Domain: youtube.com
+          ThirdParty: true
+          Description: CookieConsent::cookie_description_third_party_marketing_html
+          Expiration: 8
+          ExpirationUnit: months
+      AutoClearCookies:
+        - Name: '/^(yt-player-.*|GPS|VISITOR_INFO1_LIVE|PREF|__Secure-YEC|YSC|CONSENT|SOCS)$/'
+  IframeServices:
+    youtube:
+      EmbedUrl: 'https://www.youtube.com/embed/{data-id}'
+      #ThumbnailUrl: 'https://i3.ytimg.com/vi/{data-id}/hqdefault.jpg'
```

Comment:
Another commented-out service that I'm curious about. (Just figure we should document or remove, as appropriate).

Reply:
It's not in the default config anymore, but needs to be documented in the wiki (like all the other settings).